401-code-challenges/array-shift/array-shift.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+function insertShiftArray(array, insert){
+  if (!Array.isArray(array)){
+    return 'I need an array to put something into'}
+  else if (typeof insert !== 'number' && typeof insert !== 'string'){
+    return array;
+  }
+  else if (array.length === 0){
+    return [insert];
+  }
+    let insertedArray = [];
+    let middleFinger = Math.ceil(array.length / 2);
```

Comment:
I would probably name this something like `middlePointer` or `middleIndex` so that it is more clear what it is